src/routes/(console)/organization-[organization]/billing/removeAddress.svelte

```diff
@@ -21,6 +21,7 @@
             });
             trackEvent(Submit.OrganizationBillingAddressDelete);
             invalidate(Dependencies.ORGANIZATION);
```

Comment:
won't this be invalidated anyway due to `Dependencies.ORGANIZATIONS`' waterfall?

Reply:
updated